common_tests.sh

```diff
@@ -2,10 +2,10 @@
 
 SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
 
-export USE_TEST_REPO=true
+export ENVIRONMENT=test
 
 # Source the latest version of assert.sh unit testing library and include in current shell
-source /dev/stdin <<< "$(curl --silent https://raw.githubusercontent.com/hazelcast/assert.sh/main/assert.sh)"
+source /dev/stdin <<<"$(curl --silent https://raw.githubusercontent.com/hazelcast/assert.sh/main/assert.sh)"
```

Comment:
```suggestion
source /dev/stdin <<< "$(curl --silent https://raw.githubusercontent.com/hazelcast/assert.sh/main/assert.sh)"
```

Not sure why this changed, it's how we do it elsewhere:
https://github.com/hazelcast/backport/blob/e4641820c2341ee7f205f03aa87b322f27c60789/backport_tests#L8